cmd/pulumi-test-language/interface.go

```diff
@@ -731,7 +614,19 @@ func (eng *languageTestServer) RunLanguageTest(
 		localDependencies[pkg] = sdkArtifact
 
 		// Check that packing the SDK didn't mutate any files, but it may have added ignorable build files.
-		validations, err = compareDirectories(sdkTempDir, snapshotDir, true /* allowNewFiles */)
+		// Again we need to make a snapshot edit for this.
+		sdkSnapshotDir, err = editSnapshot(sdkTempDir, snapshotEdits)
+		if err != nil {
+			return nil, fmt.Errorf("sdk snapshot creation for %s: %w", pkg, err)
+		}
+		validations, err = compareDirectories(sdkSnapshotDir, snapshotDir, true /* allowNewFiles */)
```

Comment:
Wonder if we could get away without copying the entire snapshot here, and just do the comparison and replacement in-memory.  I don't think that needs to be done now, but might be useful for the performance of conformance tests.  May be worth adding a `TODO`?

Reply:
Yeh could do, makes the comparison a lot trickier and I figured copying files around isn't _that_ expensive relative to all codegen, and process running were doing for these tests. I'll add a comment though.